extensions/amp-list/amp-list.md

```diff
@@ -220,6 +220,13 @@ In several cases, we may need the `<amp-list>` to resize on user interaction. Fo
 </amp-list>
 ```
 
+[filter formats="websites, stories"]
+The following feature is available with the `amp-list-layout-container` [experiment](https://amp.dev/documentation/guides-and-tutorials/learn/experimental/) enabled.
```

Comment:
@CrystalOnScript Is there a more appropriate syntactic format to designate an experiment in the documentation? Also as a note, the bit about the feature being experimental only applies to websites and stories.

Reply:
There's no specific syntax, but it may be helpful to give it an alternative heading for websites and stories - maybe just `Experimental feature: dynamic resizing`. wdyt?